components/composition/Spacing/Spacing.css

```diff
@@ -0,0 +1,43 @@
+/* top */
+
+.diamond-spacing-top-xs {
+  margin-top: var(-diamond-spacing-xs);
+}
+
+.diamond-spacing-top-sm {
```

Comment:
I think we should remove top spacing, only bottom should be used in templates

Reply:
I use top quite a lot when there are conditional elements

```jsx
<diamond-card></diamond-card>
{conditional && (<diamond-card class="diamond-spacing-top-md"></diamond-card>)
```

Reply:
Or maybe not if it's a utility class and not a react thing 🤔

Reply:
Alright leave it in